### ISSUE TITLE
some kind of PYPATH problem with TrainingExample (?)

### ISSUE THREAD

**Issue by matpalm** [matpalm/drivebot#1]
when I run:
`./policy_runner.py --policy Baseline`

I get

```
Traceback (most recent call last):
File "./policy_runner.py", line 70, in <module>
rospy.Subscriber('/drivebot/training_egs', TrainingExample, call_policy_training)
NameError: name 'TrainingExample' is not defined
```

However, I checked with
`rosmsg show drivebot/TrainingExample`
it worked.
So, I was a little bit confused.

The same error applies to other policies.

I am using Ubuntu 14.04 LTS.

**Comment by dorje** [comment]
Hello Mat,

I am so sorry for this. I solved this problem.

It was because I commented out those two lines when I first time met some troubles:

from drivebot.msg import TrainingExample
from drivebot.srv import ActionGivenState

Sorry for this. Now the robots are running on my PC.

Thanks for your help.

I think I will soon have some experiment results, and I will let you know.
Oh, I have to finish a paper this week, so, maybe next week I will work on this.

Cheers,
Dorje
